packages/ui/src/description-list/DescriptionList.stories.tsx

```diff
@@ -146,3 +148,21 @@ export const nested = () => (
     />
   </DescriptionList>
 );
+
+export const payment = () => (
+  <DescriptionList>
+    <DescriptionListItem
+      icon={<CreditCard />}
+      price="$1200"
```

Comment:
Is it something new?

Reply:
removed this type of component already existed in DescriptionItem component